receiver/windowsperfcountersreceiver/config.go

```diff
@@ -46,23 +69,53 @@ func (c *Config) Validate() error {
 		errs = multierr.Append(errs, fmt.Errorf("must specify at least one perf counter"))
 	}
 
+	if len(c.MetricMetaData) == 0 {
+		errs = multierr.Append(errs, fmt.Errorf("must specify at least one metric"))
+	}
+
+	for name, metric := range c.MetricMetaData {
+		if metric.Unit == "" {
+			metric.Unit = "1"
+		}
+
+		if (metric.Sum != SumMetric{}) {
+			if metric.Sum.Aggregation != "cumulative" && metric.Sum.Aggregation != "delta" {
+				errs = multierr.Append(errs, fmt.Errorf("sum metric %q includes an invalid aggregation", name))
+			}
+		} else if (metric.Sum != SumMetric{}) && (metric.Gauge != GaugeMetric{}) {
+			errs = multierr.Append(errs, fmt.Errorf("metric %q provides both a sum config and a gauge config", name))
+		}
+	}
+
 	var perfCounterMissingObjectName bool
 	for _, pc := range c.PerfCounters {
 		if pc.Object == "" {
 			perfCounterMissingObjectName = true
 			continue
 		}
 
+		if len(pc.Counters) == 0 {
+			errs = multierr.Append(errs, fmt.Errorf("perf counter for object %q does not specify any counters", pc.Object))
+		}
+
+		for _, counter := range pc.Counters {
+			foundMatchingMetric := false
+			for name := range c.MetricMetaData {
+				if counter.Metric == name {
+					foundMatchingMetric = true
+				}
+			}
+			if !foundMatchingMetric {
+				errs = multierr.Append(errs, fmt.Errorf("perf counter for object %q includes an undefined metric", pc.Object))
+			}
```

Comment:
Would it be very difficult to allow counters without metric metadata? Basically behaving the same as before? (I think it would be correct to emit them as int gauges though)

Reply:
Added basic handling for no metrics defined